js/main.js

```diff
@@ -0,0 +1,25 @@
+const $photoURL = document.querySelector('#url');
+const $form = document.querySelector('form');
+
+$photoURL.addEventListener('input', event => {
+  const url = document.querySelector('#url').value;
+  document.querySelector('#img-preview').setAttribute('src', url);
+});
+
+$form.addEventListener('submit', event => {
+  event.preventDefault();
+  const newTitle = document.querySelector('#title').value;
+  const newUrl = document.querySelector('#url').value;
+  const newNotes = document.querySelector('#notes').value;
```

Comment:
Works, but I would probably either do `$form.elements` and grab them that way, or `document.forms.elements`. For the most part, we don't want to do queries inside a function, unless it's for an element that gets created in JS and appended later. If the element already exist in our HTML, I would query for them globally and utilize the variable. This way it prevents us from having to traverse the DOM over and over again